xml/ha_maintenance.xml

```diff
@@ -0,0 +1,375 @@
+    </para>
+   </step>
+   <step>
+    <para>
+     Set its value to <literal>No</literal> and click <guimenu>Apply</guimenu>.
+    </para>
+   </step>
+   <step>
+    <para>
+     After you have finished your maintenance task, set
+     <guimenu>is-managed</guimenu> to <literal>Yes</literal> (which is the
+     default value) and apply your changes.
+    </para>
+    <para>
+     From this point on, the resource will be managed by the &hasi; software
+     again.
+    </para>
+   </step>
+  </procedure>
+ </sect1>
+ <sect1 xml:id="sec.ha.manual_config.cli.maint.mode">
+  <title>Using Maintenance Modes with &crmsh;</title>
+
+  <variablelist>
+   <varlistentry>
+    <term>Applying Maintenance Mode to your Cluster</term>
+    <listitem>
+     <para>
+      In case you want to put the whole cluster in maintenance mode, use the
+      following command:
+     </para>
+<screen>&prompt.root;<command>crm</command> configure property maintenance-mode=true</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Applying Maintenance Mode to Nodes</term>
+    <listitem>
+     <para>
+      To put a node into maintenance mode (for example, the node
+      <literal>&node1;</literal>):
+     </para>
+<screen>&prompt.root;<command>crm</command> node maintenance &node1;</screen>
+     <para>
+      The <command>crm status</command> command will show the maintenance mode
+      for &node1; and no more resources will be allocated to that node. To
+      remove the maintenance flag from the node, use:
+     </para>
+<screen>&prompt.root;<command>crm</command> node ready &node1;</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Applying Maintenance Mode to Resources</term>
+    <listitem>
+     <para>
+      To put a specific resource into maintenance mode (for example, the
+      resource <literal>ipaddress</literal>) enter:
+     </para>
+<screen>&prompt.root;<command>crm</command> resource maintenance ipaddress true</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Putting a Resource Into Unmanaged Mode</term>
+    <listitem>
+     <para>
+      To put a specific resource into unmanaged mode, enter:
+     </para>
+<screen>&prompt.root;<command>crm</command> resource unmanage <replaceable>RESOURCE_ID</replaceable></screen>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+ </sect1>
+ <sect1>
+  <title>Rebooting a Cluster Node</title>
+  <para>If you need to reboot a cluster node for a maintenance task, it is not a
+   good idea to simply shut down the node down without precautions. Rebooting a cluster
+   node (as well as stopping the cluster stack on a node) will trigger the
+   normal &ha; recovery process&mdash;like in the case of a reboot caused by a fault.
+   That means the node will be fenced. Services that had been running on
+   this node will be restarted on other nodes (or migrate back to the same node
+   when it has finished rebooting and has rejoined the cluster).
```

Comment:
The description here is not exactly true or relevant...

Stopping/rebooting a cluster node (or the cluster service on a node) will first stop/move the resources off the node. Of course like in other cases, stop of resources might fail or time out, which would have to trigger fencing targeting the node.

Reply:
Sorry, the text above was based on part of the answers that arrived on the mailing list when I asked. ;) I will try to change it according to your suggestions.